images.go

```diff
@@ -27,12 +29,19 @@ type Image struct {
 	// value which was generated by the library.
 	ID string `json:"id"`
 
-	// Digest is a digest value that we can use to locate the image.
+	// Digest is a digest value that we can use to locate the image, if one
+	// was specified at creation-time.
 	Digest digest.Digest `json:"digest,omitempty"`
 
+	// Digests is a list of digest values of the image's manifests, and
+	// possibly a manually-specified value, that we can use to locate the
+	// image.  If Digest is set, its value is also in this list.
+	Digests []digest.Digest `json:"-"`
+
 	// Names is an optional set of user-defined convenience values.  The
 	// image can be referred to by its ID or any of its names.  Names are
-	// unique among images.
+	// unique among images, and are typically the text representation of
+	// tagged or canonical references.
```

Comment:
(FWIW libpod now has code that fails if parsing elements with `reference.ParseNormalizedNamed` fails.  I don’t have an opinion on whether it should fail like that, just that it would be nice everything agreed on whether this format is required or only recommended.)

Reply:
This is me walking back some larger changes that would have added a requirement that values passed to `SetNames()` for images also passed `ParseNormalizedNamed()`.  I remain unsure if that's something we want to be enforcing down here.